lib/packable/extensions/io.rb

```diff
@@ -10,6 +10,25 @@ def self.included(base) #:nodoc:
         base.alias_method_chain :each, :packing
       end
 
+      # Methods supported by seekable streams.
+      SEEKABLE_API = %i[pos pos= seek rewind].frozen
+
+      # Check whether can seek without errors.
+      def seekable?
+        if @seekable.nil?
```

Comment:
Must use `defined?(@seekable)` otherwise you'll get "warning: instance variable @seekable not initialized" if warnings are on.

Reply:
Ok.